build/patches/Bromite-AdBlockUpdaterService.patch

```diff
@@ -316,36 +384,50 @@ diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/AdBloc
 new file mode 100644
 --- /dev/null
 +++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/AdBlockPreferences.java
-@@ -0,0 +1,61 @@
+@@ -0,0 +1,131 @@
 +// Copyright 2015 The Chromium Authors. All rights reserved.
 +// Use of this source code is governed by a BSD-style license that can be
 +// found in the LICENSE file.
 +
 +package org.chromium.chrome.browser.settings;
 +
 +import android.os.Bundle;
++import android.content.Context;
 +import androidx.preference.Preference;
 +import androidx.preference.PreferenceFragmentCompat;
 +import org.chromium.components.browser_ui.settings.ChromeSwitchPreference;
 +
++import org.chromium.base.Log;
++import org.chromium.base.ContextUtils;
 +import org.chromium.components.embedder_support.browser_context.BrowserContextHandle;
 +import org.chromium.components.browser_ui.site_settings.SiteSettingsPreferenceFragment;
 +import org.chromium.components.browser_ui.site_settings.WebsitePreferenceBridge;
++import org.chromium.components.browser_ui.settings.TextMessagePreference;
++import org.chromium.components.browser_ui.settings.ButtonPreference;
 +import org.chromium.components.content_settings.ContentSettingsType;
 +import org.chromium.components.browser_ui.settings.SettingsUtils;
 +import org.chromium.chrome.browser.flags.CachedFeatureFlags;
++import org.chromium.chrome.browser.flags.CachedFeatureFlags.AdBlockCallback;
 +import androidx.annotation.VisibleForTesting;
++import androidx.preference.Preference.OnPreferenceClickListener;
 +import org.chromium.chrome.R;
 +
++import java.text.DateFormat;
++
 +/**
 + * Fragment that allows the user to configure AdBlock related preferences.
 + */
-+public class AdBlockPreferences extends SiteSettingsPreferenceFragment {
++public class AdBlockPreferences extends SiteSettingsPreferenceFragment
++                                implements AdBlockCallback {
 +    @VisibleForTesting
-+    public static final String PREF_ADBLOCK_SWITCH = "adblock_switch";
++    private static final String PREF_ADBLOCK_SWITCH = "adblock_switch";
 +    private static final String PREF_ADBLOCK_EDIT = "adblock_edit";
++    private static final String PREF_ADBLOCK_INDEX_VERSION = "adblock_current_index_version";
++    private static final String PREF_ADBLOCK_CHECK_NOW = "adblock_startchek";
```

Comment:
```suggestion
+    private static final String PREF_ADBLOCK_CHECK_NOW = "adblock_startcheck";
```

Reply:
fixed in https://github.com/bromite/bromite/pull/905/commits/78a9c7eb85cadf9ba849f400a646e8be3999b629